backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -28,6 +28,29 @@ include::modules/velero-obtaining-by-accessing-binary.adoc[leveloffset=+1]
 include::modules/oadp-debugging-oc-cli.adoc[leveloffset=+1]
 include::modules/migration-debugging-velero-resources.adoc[leveloffset=+1]
 
+[id="oadp-pod-crash-resource-request"]
+== Pods crash or restart due to lack of memory or CPU
+
+If a Velero or Restic pod crashes due to a lack of memory or CPU, you can set specific resource requests for either of those resources.
```

Comment:
It looks like in other parts of the doc `Velero` and `Restic` are in backtics when referring to the pods specifically, like "`Velero` pods" and "`Restic` pods"

http://file.emea.redhat.com/rhoch/resource_req/backup_and_restore/application_backup_and_restore/troubleshooting.html#oadp-backup-location-contains-invalid-directories_oadp-troubleshooting

Reply:
Fixed.